mu-plugins/global-fonts/index.php

```diff
@@ -89,9 +89,12 @@ function maybe_preload_font( $preload ) {
 		return $preload;
 	}
 
-	$combined_fonts_subsets = array_combine( (array) $style->extra['preload']['fonts'], (array) $style->extra['preload']['subsets'] );
+	$fonts = (array) $style->extra['preload']['fonts'];
+	$subsets = (array) $style->extra['preload']['subsets'];
```

Comment:
`array_combine` would fail the case:
`global_fonts_preload( 'Inter, Inter', 'Latin, Latin-ext' );` as the key (i.e., `'Inter'`) can't be duplicated.